packages/frames.js/src/render/farcaster/frames.tsx

```diff
@@ -10,11 +10,14 @@ import {
 import { FrameButton } from "../../types";
 import { FrameActionBodyPayload, FrameContext } from "../types";
 import { FarcasterSignerState } from "./signers";
+import { hexToBytes } from "viem";
 
 export interface FarcasterFrameActionBodyPayload
   extends FrameActionBodyPayload {}
 
 export type FarcasterFrameContext = {
+  /** Connected address of user, only sent with transaction data request */
+  address?: string;
```

Comment:
wouldn't it make sense to rename this to `connectedAddress` so it has the same name as in `FrameActionDataParsed`?